eodag/resources/providers.yml

```
@@ -5928,3 +5928,43 @@
         <<: *day_month_year
     GENERIC_PRODUCT_TYPE:
       dataset: '{productType}'
+---
+!provider # MARK: fedeo_ceda
+  name: fedeo_ceda
+  priority: 0
+  description: CEDA Federation of Earth Science Data Centres via FedEO STAC Catalog
+  roles:
+    - host
+  url: https://fedeo.ceos.org/
+  search: !plugin
+    type: StacSearch
+    api_endpoint: 'https://fedeo.ceos.org/search'
+    need_auth: false
+    ssl_verify: true
```

```suggestion
    ssl_verify: false
```
There is a SSL certificate error now on FedEO ....